src/Arcus.Messaging.Tests.Workers.ServiceBus.Queue/Program.cs

```diff
@@ -36,7 +37,9 @@ public static void Main(string[] args)
                             .UsingAuthenticationKey(eventGridKey)
                             .Build();
                     });
-                    services.AddServiceBusQueueMessagePump<OrdersMessagePump>(configuration => configuration["ARCUS_SERVICEBUS_CONNECTIONSTRING"]);
+                    services.AddServiceBusQueueMessagePump<OrdersMessagePump>(configuration => configuration["ARCUS_SERVICEBUS_CONNECTIONSTRING"])
```

Comment:
I presume we don't need the message pump type here anymore?
```suggestion
                    services.AddServiceBusQueueMessagePump(configuration => configuration["ARCUS_SERVICEBUS_CONNECTIONSTRING"])
```

Reply:
No, we _do_ need that for the registration of the message pump.
The other calls registers the handlers.

Reply:
We need the Service Bus message pump, but that's it, the message handler is implementation specific.

Reply:
That's the goal of #35

Reply:
Yes, I think I get the goal, only not sure how this relates to this.
There's nothing changed on the way the message pump is added/registered, only the handlers are now processing the messages instead of the message pump.
So, we need to register both the pump and the handlers.

Reply:
What I expected to see would be:
```csharp
services.AddServiceBusQueueMessagePump(configuration => configuration["ARCUS_SERVICEBUS_CONNECTIONSTRING"])
        .WithMessageHandler<Order, OrdersMessageHandler>();
```

What would the `messageContext => true` be for? Filtering? If so, I'd leave that out for now but that's my personal opinion.

Something we should provide is a `WithFallbackMessageHandler` in case the above flow doesn't find a matching handler, but that might be more interesting for later.

Reply:
That way we have:
1) A message pump which checks the entity for new messages and triggers corresponding message handler
2) A message handler who takes action on the deserialized message